packages/neutrino/bin/start.js

```diff
@@ -1,31 +1,50 @@
 const { start } = require('../src');
 const ora = require('ora');
+const { lookup } = require('dns');
+const { hostname } = require('os');
+const Future = require('fluture');
+
+const whenIpReady = Future.node((done) => {
+  const host = hostname();
+  lookup(host, done);
+});
 
 module.exports = (middleware, options) => {
   const spinner = ora('Building project').start();
 
-  return start(middleware, options)
-    .fork((errors) => {
-      spinner.fail('Building project failed');
-      errors.forEach(err => console.error(err));
-      process.exit(1);
-    }, (compiler) => { // eslint-disable-line consistent-return
-      if (!compiler.options.devServer) {
-        return spinner.succeed('Build completed');
-      }
+  return whenIpReady
+    .chain(ip => Future.of(process.env.HOST = process.env.HOST || ip))
```

Comment:
I'm not literally talking about `x.x.x.x`. I think we still need a way to easily pass an option from the command line without env vars. I need to think on that.